scripts/format-all.sh

```
@@ -0,0 +1,68 @@
+        echo "⚠ gofmt not found, skipping Go formatting"
+    fi
+else
+    echo "⚠ Go implementation directory not found, skipping Go formatting"
+fi
+echo ""
+
+# Run TypeScript formatting
+echo "--- Formatting TypeScript Code ---"
+if [ -d "lambdas/pstn-transfer-ts" ]; then
+    if command -v npm &> /dev/null; then
+        cd lambdas/pstn-transfer-ts
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)